specification/security/resource-manager/Microsoft.Security/stable/2019-08-01/iotSecuritySolutions.json

```diff
@@ -378,6 +378,9 @@
           "type": "string",
           "description": "Workspace resource ID"
         },
+        "additionalWorkspaces": {
```

Comment:
New properties added to the response of an API is considered a breaking change and requires a new API version:

https://microsoft.sharepoint.com/:w:/t/azureresourcemanagerteam/EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw?e=xEcppo

Reply:
Our SDK and REST docs are not published yet (will be in the near future), so customers aren't supposed to make calls to this API yet.